client-js/test/spec/tsTestSpec.ts

```diff
@@ -0,0 +1,61 @@
+import {Agent} from '../test/helloWorld';
+import {Observable} from '../app/scripts/observable';
+import {Lifetime} from '../app/scripts/lifetime';
+
+
+describe("Hello world", function() {
+  var agent;
+
+  beforeEach(function() {
+    agent = new Agent();
+  });
+
+  it("says hello", function() {
+    expect(agent.helloWorld()).toEqual("Hello world!");
+  });
+});
+
+/*
+* Test to create new observable and add callback
+* This is mostly to see if the plumbing for the
+* tests interfaces with the scripts in /app
+*/
+
+describe("observable", function() {
```

Comment:
So I can't put the ```expect().toEqual()``` in an actual callback or Jasmine complains. I can do something pretty close to what you suggest though